contact_app.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import json
+
+class JSONWriter:
+    def __init__(self):
+        pass
+    def add_contact(self,person):
+        dict = {person.get_phone_number():{"name":person.get_name(),"email":person.get_email_id(),"city":person.get_city()}}
```

Comment:
expose a method toJSON() in Person class which returns the Person as a dict, so that it can be reused at multiple places